java-frontend/src/main/java/org/sonar/plugins/java/api/tree/BreakStatementTree.java

```diff
@@ -43,6 +43,12 @@
   @Nullable
   IdentifierTree label();
 
+  /**
+   * @since 5.12
```

Comment:
I would also add a reference to the Java 12 new feature in the javadoc, to properly explain what it is.